eng/pipelines/checkout-windows-task.yml

```diff
@@ -0,0 +1,17 @@
+# Shallow checkout sources on Windows
+
+# Git path based on https://github.com/microsoft/azure-pipelines-agent/blob/f98a479ab3cc323ff17b8cb75d5ab5f953f342cc/src/Agent.Plugins/GitCliManager.cs#L90
+steps:
+  - checkout: none
+
+  - script: |
+      @echo on
+      set PATH=$(Agent.HomeDirectory)\externals\git\cmd;%PATH%
```

Comment:
should we `set PATH` before we `@echo on`? It's a little noisy in the logs. I don't have a strong opinion though. https://dev.azure.com/dnceng/public/_build/results?buildId=1853334&view=logs&j=e50db95a-e076-59fe-00be-830ac89b2894&t=566fbe2c-093e-5a7a-f419-05d4d1898380&l=13